utils/mgmt_system/__init__.py

```diff
@@ -74,6 +74,7 @@ def connect_direct_lun_to_appliance(self, vm_name, disconnect):
             disk.set_format("raw")
             disk.set_lun_storage(storage)
             disk.set_shareable(True)
+            disk.set_read_only(readonly)
```

Comment:
I think given the info we received, we can put a check here for the provider version, which should be accessible through the API.